strings/brute_force_string_searching.cpp

```diff
@@ -9,6 +9,7 @@
 #include <cstring>
 #endif
 #include <vector>
+using namespace std;
```

Comment:
Please do not use `using namespace std;`. This is what causes the CI (continuous integration) to fail.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable